server/controllers/crm/crm4Controller.test.ts

```diff
@@ -116,105 +116,4 @@ describe('CRM4 Controller', () => {
       backUrl: '/search-eform',
     })
   })
-
-  it('should set the back URL correctly when navigating back', async () => {
```

Comment:
Removed this test case as the code was flagged as duplicated code by Intellij. Also this test case is already covered in  `crmBackLink.test.ts`